CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+	
+	fire_only_once = yes
+	
+	trigger = {
+		tag = ITA
+		year = 1989
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship
+		is_vassal = no
+		war = no
+		
+		money = -150000
+		
+		#Croatian War of Independence
+		YUG = { 
+			exists = yes
+			
+			is_vassal = no
+			war_with = CRO
+		}
+		
+		CRO = { 
+			exists = yes
+			
+			is_vassal = no
+			war_with = YUG
+		}
+		
+		has_country_flag = ITA_delle_Chiaie
+		has_global_flag = ITA_CRO_Ustasha_funding
+    	has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { days = 1 }
+
+	option = {
+    	name = "Support the Croatian Government!"
+		prestige = 25
+		badboy = 2
+		
+		money = -100000
+		
+		#Domestic effect
+		any_pop = { 
+			limit = { is_primary_culture = yes }
+		    scaled_consciousness = { factor = -2 ideology = nationalist }
+			scaled_militancy = { factor = -2 ideology = nationalist }
+		}
+		
+		#Aid to Croatia
+		CRO = {
+			money = 100000
+			small_arms = 1000
+			weaponry = 1000 
+			aeroplanes = 1000
+			ammunition = 1000
+			automobiles = 1000
+			
+			add_country_modifier = { name = post_war_austerity duration = 3650 }
+		}
+		
+		#Gain relations
+		relation = { who = CRO value = 200 }
+		diplomatic_influence = { who = CRO value = 100 }
+		create_alliance = CRO
+		
+		#Lose relations with Yugoslavia
+		random_country = {
+			limit = { tag = YUG is_vassal = no }
+			relation = { who = THIS value = -100 }
+			THIS = { diplomatic_influence = { who = YUG value = -100 } }
+		}
+		
+		#Lose relations with Slovenia
+		random_country = {
+			limit = { tag = SLO is_vassal = no }
+			relation = { who = THIS value = -50 }
+			THIS = { diplomatic_influence = { who = SLO value = -50 } }
+		}
+		
+		ai_chance = { factor = 0.95 }
+	}
+	
+	option = {
+    	name = "There is no need to interfere"
+		relation = { who = CRO value = -50 }
+		diplomatic_influence = { who = CRO value = -50 }
+		ai_chance = { factor = 0.05 }
+	}
+
+}
+
+#Muggia agreement - 1
+country_event = {
+	id = 8020439
+	title = "The Muggia Agreement activation"
+	desc = EVT_8020439_DESC
+	picture = "nwo2_free_city_of_trieste"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = CRO
+		year = 1992
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship
+		is_vassal = no
+		war = no
+		
+		#ITA exists
+		ITA = {
+			exists = yes
+			is_vassal = no
+			war = no
+		}
+		
+		ITA = { has_country_flag = ITA_CRO_Koper } 
+		has_global_flag = ITA_CRO_Ustasha_funding
+		has_global_flag = ITA_Operation_Tora_Tora 
+	}
+
+	mean_time_to_happen = { months = 1 }
+	
+	immediate = { clr_global_flag = ITA_CRO_Ustasha_funding ITA = { clr_country_flag = ITA_CRO_Koper } }
+
+	option = {
+		name = "Stick to the pacts"
+		prestige = 10
+		
+		random_country = {
+			limit = { tag = ITA is_vassal = no }
+			relation = { who = THIS value = 100 }
+		}
+		
+		SLO = { 3345 = { secede_province = ITA change_province_name = "Capodistria" } } #Koper, Istria
```

Comment:
There must be a check in the triggers for SLO owning 3345

Reply:
How can I make it so that it checks for whoever owns 3345? Also sure will add a reaction event

Reply:
You can use random_country = { limit = { is_vassal = no war = no owns = 3345 } country_event = (reaction event) }